### PR TITLE
Prompt Node Retries Release

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

File: fern/docs/content/change-log/2024-02.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Prompt Node Retries
+*February 1st, 2024*
+
+Prompt Nodes in workflows now emit a selectable output for instances of when it errors. Users could use this to now build Vellum Workflow-native retry functionality for your prompts!
+
+Check out the video below for an example of how this works:
+
+https://www.loom.com/share/770a1ac0bbaa4f9d9192d336eb0a191c
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-02.mdx:8]: gonna try out a few ideas here before shippin'

File: fern/docs/content/help/workflows/common-architectures.mdx [hunk 2]
```diff
@@ -50,3 +50,15 @@ Here are the steps involved in this example:
     4. The current `prompt_output`, using a fallback mapped to a Text input variable of an empty string.
 - Step 4: The templating node feeds into a conditional node, that calls a Prompt node when the prompt's execution counter is less than the length of the array. It outputs to a terminal node in the else branch.
 - Step 5: The Prompt node receives the edge from the conditional node's IF branch, and feeds back to the templating node with its output.
+
+## 4. Retrying a Prompt Node in case of non-deterministic failure
+
+Prompt nodes support two selectable outputs - one from the model in case of a valid output and one in case of a non deterministic error. Model hosts fail for all sorts of reasons that include timeouts, rate limits, or server overload. You could make your production-grade LLM features resilient to these features by adding retry logic into your Workflows!
+
+![Workflow Retry Example](https://storage.googleapis.com/vellum-public/help-docs/workflow_retry_example.png)
```
Comment (dvargas92495) [inline fern/docs/content/help/workflows/common-architectures.mdx:58]: already in GCS

**Review by dvargas92495 — COMMENTED** [review]
Feel free to make copy suggestions directly on the PR

**Comment by dvargas92495** [comment]
Woah! branch preview for docs, we could remove from the `-branch` workflow

File: fern/docs/content/change-log/2024-02.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Prompt Node Retries
+*February 1st, 2024*
+
+Prompt Nodes in workflows now emit a selectable output for instances of when it errors. Users could use this to now build Vellum Workflow-native retry functionality for your prompts!
+
+Check out the video below for an example of how this works:
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-02.mdx:6]: Below is a suggested rewording. Here are some guidelines that might help in writing future updates like this: https://www.notion.so/vellum-ai/Product-Updates-a72a4a34446843b897fbff713e1450e1

```suggestion
You can now detect when a Prompt Node within a Workflow errors by using a Conditional Node. You can also optionally check to see what type of error it is. Using all of this, you can now build out retry logic around Prompt Nodes within your Workflow! This is useful if you want to catch retryable errors (like rate limit errors from an LLM provider) and try making the call to the LLM again.

See a demo of it in action here:
```

**Review by noanflaherty — APPROVED** [review]
🔥

File: fern/docs/content/change-log/2024-02.mdx (hunk 3 above)
Comment (dvargas92495) [reply fern/docs/content/change-log/2024-02.mdx]: God tier reference doc

**Pull request merged**